ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -77,3 +77,34 @@
 #         on_device=on_device,
 #         x=x[0],
 #     )
+
+# flatten
+# @handle_frontend_test(
+#     fn_tree="mindspore.ops.function.nn_func.flatten",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("valid"),
+#         num_arrays=1,
+#         shared_dtype=True,
+#         min_value=2,
+#         max_value=5,
+#         min_dim_size=4,
+#         shape=st.lists(st.integers(min_value=2, max_value=10), min_size=3, max_size=3),
```

Comment:
@Sam-Armstrong 
1. shared_dtype: If the shared_dtype parameter is set to True, it indicates that the arrays in your test cases will all have the same data type (`dtype`). This can be useful for checking how well the `flatten` function handles arrays of consistent data types.

i think shape can be removed here